llvm/test/Transforms/SimplifyCFG/simplify-cfg-unroll.ll

```diff
@@ -0,0 +1,939 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
```

Comment:
This should be a LoopUnroll test, not a SimplifyCFG test.

Reply:
ok,  let me change it?